todo_app/app.py

```diff
@@ -1,11 +1,27 @@
 from flask import Flask
-
+from flask import render_template
+from flask import request
+from flask import redirect
```

Comment:
Nothing wrong with this, but you may find that it's nicer to combine these together into a single line:
```suggestion
from flask import render_template, request, redirect
```
(And similarly for the `session_items` imports)